src/main/java/mate/academy/spring/validator/PasswordValidator.java

```diff
@@ -4,13 +4,14 @@
 import javax.validation.ConstraintValidatorContext;
 import mate.academy.spring.dto.request.UserRegistrationDto;
 
-public class PasswordValidator implements ConstraintValidator<RepeatPassword,
+public class PasswordValidator implements ConstraintValidator<ValidPassword,
         UserRegistrationDto> {
     @Override
     public boolean isValid(UserRegistrationDto dto, ConstraintValidatorContext context) {
         String password = dto.getPassword();
         String repeatPassword = dto.getRepeatPassword();
-        if (password == null || repeatPassword == null) {
+        if (password == null || repeatPassword == null
+                || password.length() < 8 || repeatPassword.length() < 8) {
```

Comment:
please create a constant for hardcoded numbers.
do we really need to check the repeatPassword length here since we comparing password and repeatPassword in line 17?

Reply:
I definitely agree, it was due to speed and inattention, I'll fix it now and keep it in mind for the future